packages/cli/src/cmds/preview/previewEnv.ts

```diff
@@ -131,7 +132,7 @@ export default class PreviewEnv extends YargsCommand {
 
     cliTelemetry.withRootFolder(workspaceFolder);
     this.telemetryProperties[TelemetryProperty.PreviewType] =
-      env.toLowerCase() === environmentManager.getLocalEnvName() ? "local" : `remote-${env}`;
+      environmentNameManager.isRemoteEnvironment(env.toLowerCase()) ? `remote-${env}` : env;
```

Comment:
`env.toLowerCase()`, or will `env` contains upper case?

Reply:
Same for all telemetry changes

Reply:
Core doesn't treat env specially, it just rely on OS file system behavior.
https://github.com/OfficeDev/TeamsFx/blob/fef761eba616243b7cff601080407af6878702fe/packages/fx-core/src/component/utils/pathUtils.ts#L53
So on Windows it's case-insensitive. Probably case sentive on macOS if using case-sensitive file system.

But CLI preview treats env name as case-insensitive despite of OS.

Core and CLI are already inconsistent. 

The goal of this PR is not to solve this existing problem. Just be consistent with current impl (in core don't do conversion, in cli `toLowerCase()`)

Reply:
Agreed to be consistent with current impl.
But this line's change uses `env` instead of `"local"`. Is it possible to send "**LoCal**" (any case) as telemetry? Will that impact telemetry query?